docker-compose.dev.yml

```diff
@@ -1,22 +1,45 @@
-version: "3.7"
+# Docker Compose project for local development
+name: ipl-dagster-pipeline
+
+# Define placeholder for running a container with the same UID/GID as your local user
+x-local-user: &local-user ${DOCKER_LOCAL_USER:?Variable needs to be set in .env (e.g. "DOCKER_LOCAL_USER=1000:1000")}
 
 services:
-  # This service runs the postgis DB used to publish datasets, either via GeoServer or REST services,
-  # and to perform data transformation
-  ipl_postgis:
-    image: postgis/postgis:${IPL_POSTGIS_VERSION_TAG}
-    container_name: ipl_postgis
+  python:
+    image: ipl-dagster-pipeline:local-dev
```

Comment:
This will fail, if no such image exists, no? If it needs to be built in advance, the README should note this.

Reply:
Added the build command and other stuff to the readme